.github/workflows/branch-cleanup-orchestrator.yml

```
@@ -0,0 +1,142 @@
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        
+    - name: Configure Git
+      run: |
+        git config --global user.name "Branch Cleanup Bot"
+        git config --global user.email "bot@kindlemint.ai"
+        
+    - name: Set up Python
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)